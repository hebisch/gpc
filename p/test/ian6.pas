program cpptest(input,output);

begin

#ifdef __GPC__
        writeln('#ifdef __GPC__');
#endif

#if defined (__GPC__)
        writeln('defined (__GPC__)');
#endif
end.
