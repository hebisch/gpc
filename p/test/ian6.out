#ifdef __GPC__
defined (__GPC__)
